webapp/src/client.ts

```diff
@@ -4,10 +4,10 @@
 import {AnyAction, Dispatch} from 'redux';
 import qs from 'qs';
 
-import {GetStateFunc} from 'mattermost-redux/types/actions';
+import {GetStateFunc, batchActions} from 'mattermost-redux/types/actions';
```

Comment:
We can probably remove this new import, right?
```suggestion
import {GetStateFunc} from 'mattermost-redux/types/actions';
```

Reply:
Sure, will do in the next commit.